### ISSUE TITLE
Extension shows install.html on Chrome update

### ISSUE THREAD

**Issue by keif888** [andyvalerio/photoprism-geolocation#12]
Hi Andy,
Looks like I made a mistake in worker.js.
This isn't breaking the extension, it just means that the install.html shows up every time the browser is updated.  

The following code, right at the start of the script, is firing every time Chrome is updated and loading the install.html, when it should filter to only when the extension is installed.  
https://github.com/andyvalerio/photoprism-geolocation/blob/a642bd5ed75e72a5ad7d08eef2c05c3d213b5e91/extension/worker.js#L3
```
chrome.runtime.onInstalled.addListener(async ({ reason }) => {
    // Show the install tab so the user knows they have to add their PhotoPrism server.
    chrome.tabs.create({ url: "install.html" });
});
```

This should probably be the following (assuming that you do not want install.html to show in upgrades):
```
chrome.runtime.onInstalled.addListener(async ({ reason }) => {
    if (reason !== 'install') {
      return;
    }
    // Show the install tab so the user knows they have to add their PhotoPrism server.
    chrome.tabs.create({ url: "install.html" });
});
```

See https://developer.chrome.com/docs/extensions/reference/api/alarms#set-alarm for example.
Unlike every other example from google that I have seen which doesn't have the check in it.

**Comment by andyvalerio** [comment]
Yeah I've been meaning to fix this for a few weeks as it really bothers me every time the browser is updated. 🤣

I'll see if I can push the change this weekend and release it. Thanks for looking into it @keif888!

**Comment by andyvalerio** [comment]
Fixed now in version 2.1 soon out in the store.
